ast/compile.go

```diff
@@ -113,6 +113,8 @@ type Compiler struct {
 	inputType             types.Type                    // global input type retrieved from schema set
 	annotationSet         *AnnotationSet                // hierarchical set of annotations
 	strict                bool                          // enforce strict compilation checks
+	keepModules           bool                          // whether to keep the unprocessed, parse modules (below)
+	modules               map[string]*Module            // parsed, but otherwise unprocessed modules, kept track of when keepModules is true`
```

Comment:
The near name shadowing of `Modules` vs `modules` (raw version) is interesting. Not sure what a better name or naming scheme would be, since `rawModules` implies they haven't been parsed from strings yet. :thinking:

Reply:
I've renamed it -- `parsedModules` pretty much mirrors `ParsedModules()`. BTW I didn't want to just expose the map (like done with `Modules`) because hiding it behind an accessor func gives us some more leeway in terms of what's happening beneath the func.